Source/JavaScriptCore/bytecode/InlineCacheCompiler.cpp

```diff
@@ -439,9 +441,13 @@ void InlineCacheCompiler::restoreScratch()
 void InlineCacheCompiler::succeed()
 {
     restoreScratch();
-    if (m_jit->codeBlock()->useDataIC())
-        m_jit->farJump(CCallHelpers::Address(m_stubInfo->m_stubInfoGPR, StructureStubInfo::offsetOfDoneLocation()), JSInternalPtrTag);
-    else
+    if (m_jit->codeBlock()->useDataIC()) {
+        if (JITCode::isBaselineCode(m_jit->codeBlock()->jitType())) {
+            InlineCacheCompiler::emitDataICEpilogue(*m_jit);
```

Comment:
Why do you need the `InlineCacheCompiler::` here? Or is that just a copy paste issue?

Reply:
Fixed.